samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -165,14 +165,28 @@ Location tracking
 All matters concerning location tracking are handled in the :file:`src/location_tracking.c` file.
 For the most part, this amounts to setting up a periodic location request, and then passing the results to a callback configured by the :file:`src/application.c` file.
 
-Additionally, the :ref:`lib_location` library requires GPS assistance data (either A-GPS, P-GPS, or both) to have the fastest possible `time-to-first-fix <TTFF_>`_.
+For GNSS location tracking, the :ref:`lib_location` library requires GPS assistance data (either A-GPS, P-GPS, or both) to have the fastest possible `time-to-first-fix <TTFF_>`_.
 The library requests this data automatically, but has no way to automatically receive the response.
 Therefore, all MQTT messages received from `nRF Cloud`_ must be be passed directly through to the :ref:`lib_nrf_cloud_agps` and :ref:`lib_nrf_cloud_pgps` libraries to be checked for A-GPS and P-GPS data, respectively.
 The :c:func:`location_assistance_data_handler` function handles this task, and MQTT messages are sent directly to it from the :file:`src/connection.c` file.
 
+For location readings to be visible in the nRF Cloud portal, they must be marked as enabled in the `Device Shadow <nRF Cloud Device Shadows_>`_.
+This is performed by the :file:`src/connection.c` file.
+
+Each enabled location method is tried in the following order until one succeeds: GNSS, Wi-Fi, then Cellular.
+
+The GNSS and Cellular location tracking methods are enabled by default and will work on both Thingy:91 and nRF9160 DK targets.
+
+.. _nrf_cloud_mqtt_multi_service_wifi_location_tracking:
+
+The Wi-Fi location tracking method is not enabled by default and requires an nRF9160 DK with an attached nRF7002 EK Wi-Fi evaluation kit.
+
+When enabled, this location method will scan nearby access points MAC addresses, and submit them to nRF Cloud to obtain a location estimate.
+
+See :ref:`_nrf_cloud_mqtt_multi_service_wifi_location_tracking` for details on how to enable Wi-Fi location tracking.
```

Comment:
This link does not really make any sense as it points back to this same section. Suggesting to leave it out.

Reply:
Yeah, I screwed up with the link formatting, but it actually points to a different, albeit similarly named section. I will correct the link, but leave it, if that is alright with you

Reply:
No worries, just make sure the link format is correct and points to a section you intended to.